tests/api/admin-notes.php

```diff
@@ -139,6 +139,23 @@ public function test_get_warning_notes() {
 		$this->assertEquals( $notes[0]['title'], 'PHPUNIT_TEST_NOTE_2_TITLE' );
 	}
 
+
+	/**
+	 * Test getting notes of a certain status.
+	 */
+	public function test_get_actioned_notes() {
```

Comment:
Thanks for the test 🌟 - should we add a test case for requesting an invalid action too?

Reply:
Done in https://github.com/woocommerce/wc-admin/pull/1763/commits/279818019b54f63a5302eba46bb348cbd5f24ce6.